samples/template-operator/operator/controllers/sample_controller.go

```diff
@@ -117,3 +127,13 @@ func (m *ManifestResolver) Get(obj types.BaseCustomObject) (types.InstallationSp
 		},
 	}, nil
 }
+
+// TemplateRateLimiter implements a rate limiter for a client-go.workqueue.  It has
+// both an overall (token bucket) and per-item (exponential) rate limiting.
+func TemplateRateLimiter(failureBaseDelay time.Duration, failureMaxDelay time.Duration,
```

Comment:
Lets introduce a `RateLimitArgs` flag here

Reply:
QQ: Do you want a flag which turns off the rate limiting completely? Since all input values for the rate limtier are already configured by flags. Have a look here: https://github.com/kyma-project/lifecycle-manager/pull/214/files#diff-f7a109f64839bd112a58f412858bd1543effde751d03b04028d0d8a330d5d2b4R124-R131

Reply:
Im just talking about a struct to pass to TemplateRateLimiter so that you do not have to pass 4 args into setupWithManager